.github/ISSUE_TEMPLATE/ui-copy-request.yaml

```
@@ -0,0 +1,97 @@
+name: "UI copy request"
+description: Submit a UI copy-related request to the security documentation team.
+title: "[UI copy]: "
+labels: "ui-copy"
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Hello! Use this form to submit a UI-related issue to the [@elastic/security-docs](https://github.com/orgs/elastic/teams/security-docs) team.
+  - type: textarea
+    id: description
+    attributes:
+      label: Description
+      description: Summarize your request here. What is the work and how can we help? Do we need to write new copy, or edit existing copy? Feel free to include screenshots.
+      placeholder: |
+        What: We're introducing new feature A.
+        When: This feature will launch at the completion of project B.
+        Why: This feature will make X, Y, and Z easier for the user.
+    validations:
+      required: true
+  - type: textarea
+    id: related
+    attributes:
+      label: Related links / assets
+      description: Where can the documentation team learn more about this feature?
+      value: |
+        Please include each of the following, if applicable:
+        Figma link(s):
+        Github epic link(s):
+        Github issue link(s):
+        How to find the text in a production environment:
+        Testing environment information:
+        NOTE: To keep sensitive information such as credentials secure, do not include it in this form.
+    validations:
+      required: true
+  - type: dropdown
+    id: doc-set
+    attributes:
+      label: Which documentation set does this change impact?
+      description: Stateful, Serverless, or both?
```

For consistency with our other template, changing stateful -> ESS. 
```suggestion
      description: ESS, serverless, or both?
```